### PR TITLE
Js blood notifications

### PR DESCRIPTION
**Pull request by hannnegoossens**

### CODE REVIEW

File: src/js/tester.js [hunk 1]
```diff
@@ -38,10 +39,12 @@
 					type: 'POST',
 					url : 'https://fcm.googleapis.com/fcm/send',
 					contentType : 'application/json',
-					data : JSON.stringify(msg),
-					dataType : 'json',
-					succes : function(data) {
-						console.log(data);
+					data : JSON.stringify(notification),
+					success : function(data) {
+						console.log("Success!");
```
Comment (thysruben) [inline src/js/tester.js]: Something like [this](https://www.webcomponents.org/element/PolymerElements/paper-toast/paper-toast) could be nice for letting user know the notification was sent?

Comment (hannnegoossens) [reply src/js/tester.js]: I know, I'm ermee bezig! :)

Comment (thysruben) [reply src/js/tester.js]: Awesome! Great work so far 👌

**Comment by hannnegoossens** [comment]
- [x] Bug: body is undefined when only blood group and title are filled in
- [x] Make sure notifications are sent (/topics/to)

**Review by thysruben — APPROVED** [review]
👍

**Pull request merged**